tools/debian/rules

```diff
@@ -38,9 +38,7 @@ override_dh_install:
 	rm debian/tmp/etc/motd.d/cockpit debian/tmp/etc/issue.d/cockpit.issue
 
 	# unpackaged modules
-	for m in kdump selinux; do rm -r debian/tmp/usr/share/cockpit/$$m; done
-	rm debian/tmp/usr/share/metainfo/org.cockpit-project.cockpit-kdump.metainfo.xml
-	rm debian/tmp/usr/share/metainfo/org.cockpit-project.cockpit-selinux.metainfo.xml
+	cd debian/tmp/usr/share; for m in kdump selinux; do rm -r cockpit/$$m metainfo/org.cockpit-project.cockpit-$$m.metainfo.xml; done
```

Comment:
ugh.  Okay.  This is a Makefile.  This is fine.  But it sure looks awful :rofl:

Reply:
Honestly, can you just take this in the other direction and delete all 4 files explicitly with full path names?

Reply:
Sure! done.